.github/workflows/build-system-test-react-native.yml

```diff
@@ -0,0 +1,187 @@
+# Description: this workflow runs build mega-apps that have all the connected components for React Native.
+
+name: Build System Test Canary / React Native
+
+permissions:
+  id-token: write # This is required for aws-actions/configure-aws-credentials
+
+on:
+  schedule:
+    - cron: '0 */1 * * *' # Run at the first minute of every hour
+
+jobs:
+  build:
+    runs-on: macos-latest
+    environment: ci
+    strategy:
+      fail-fast: false
+      matrix:
+        framework: [react-native]
```

Comment:
Do we need this `matrix` if this workflow only tests `react-native`?

Reply:
Because we use it on line 26 for `platform: [ios, android]` 👻

Reply:
Oh yeah, sorry my question wasn't worded well. I was talking more about `matrix.framework`. But I guess we're doing this for abstraction later?

Reply:
Oh, yea, you are right. That's the legacy from separating RN from other frameworks. `framework` in matrix is no longer needed since it's the only framework in this workflow.

Reply:
Done 3e1065e35